pkg/provider/azure_managedDiskController.go

```diff
@@ -67,6 +67,8 @@ type ManagedDiskOptions struct {
 	DiskIOPSReadWrite string
 	// Throughput Cap (MBps) for UltraSSD disk
 	DiskMBpsReadWrite string
+	// Logical sector size in bytes for Ultra disks
+	LogicalSectorSize int
```

Comment:
since `diskProperties.CreationData.LogicalSectorSize` is int32, what about set it as `int32`, then you don't need to convert

Reply:
Done